README.md

```diff
@@ -195,7 +206,41 @@ Vaultenv will make the following environment variables available:
  - `FOOBAR` with the contents of the `foobar` field of the secret at
    `production/third-party`.
 
-## Allowed characters
+### Behavior configuration
+
+Vaultenv supports loading behavior configuration from files (in addition to the
+CLI flags and environment variable lookups). Vaultenv currently looks for these
+files in the following places:
+
+ - `/etc/vaultenv.conf`
+ - `$HOME/.config/vaultenv/config.conf`
+ - `$CWD/.env`
+
+These config files support the exact same syntax as the environment variables
+that you can use to configure Vaultenv. See the `--help` output for a list of
+what's available.
+
+These files follow the `.env` format (as popularized by [this Ruby
+gem](https://github.com/bkeepers/dotenv)). An example:
+
+```
+# /etc/vaultenv.conf
+# Also: comments are allowed if they start with `#`.
+VAULT_TOKEN="your-vault-token"
+VAULT_PORT="8200"
+VAULTENV_INHERIT_ENV="yes"
+```
+
+This is mostly useful for use on devmachines. It allows you to:
```

Comment:
Nit: s/devmachines/development machines (since I'm not sure if everyone uses this terminology)